pkg/synchromanager/clustersynchro_manager.go

```diff
@@ -282,8 +292,18 @@ func (manager *Manager) reconcileCluster(cluster *clusterv1alpha2.PediaCluster)
 	}
 
 	manager.UpdateClusterSynchroCondition(cluster.Name, clusterv1alpha2.RunningConditionReason, "", metav1.ConditionTrue)
+	syncResources := cluster.Spec.SyncResources
+	if templateName := cluster.Spec.ClusterSyncResources; templateName != "" {
+		template, err := manager.clusterSyncResourcesLister.Get(templateName)
+		if err != nil && !apierrors.IsNotFound(err) {
+			klog.ErrorS(err, "Failed to get cluster sync template", "cluster", cluster.Name)
```

Comment:
Usually Error means that the program is abnormal and needs to end the subsequent logic

It is recommended to use `Warn` and add the TODO that we discussed last time to do in the future